portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -54,7 +61,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         response.sendRedirect("/comments.html");
     }
     private Comment getComment(HttpServletRequest request) {
```

Comment:
This function seems like it does more than just getting a `Comment`

Reply:
I would take a look at [this](https://scotch.io/bar-talk/s-o-l-i-d-the-first-five-principles-of-object-oriented-design) article on the SOLID principles